content/agent/_index.md

```diff
@@ -64,7 +64,13 @@ To automatically transition between agent configuration paths and formats from A
 
 `sudo -u dd-agent -- datadog-agent import`
 
-The command parses an existing `datadog.conf` and convert all the bits that the new Agent still supports to the new `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.
+The command parses an existing `datadog.conf` and convert all the bits that the new Agent still supports to the new `datadog.yaml` configuration file. It also copies configuration files for checks that are currently enabled.  
+
+For Mac and Windows environment use:  
+
+`datadog-agent import <old_configuration_dir> <destination_dir>` 
```

Comment:
I think we should explain that `old_configuration_dir` is the directory containing the `datadog.conf` file and `destination_dir` the file where we will write the imported `datadog.yaml` (should be the same as `old_configuration_dir` on both windows & osx).